etc/PlayOnLinux.desktop

```diff
@@ -4,5 +4,5 @@ Name=PlayOnLinux
 Comment=Front-end application for the wine
 Type=Application
 Exec=playonlinux %F
-Icon=/usr/share/playonlinux/etc/playonlinux.png
+Icon=/usr/share/pixmaps/playonlinux.png
```

Comment:
Is there any reason for using a hardcoded path instead of just "playonlinux" for the icon name?